definitions/scenario/scenarios.yaml

```diff
@@ -72,7 +72,7 @@
 # EMF-ECEMF collaboration scenarios
 - EMFECEMF_NZero
 
-# ECEMF - Work package 5 task 5.1 trade limitation scenarios
+# ECEMF - Work package 5 task 5.1 trade limitation scenarios (<= can be removed in the final iteration, but keep it for now)
```

Comment:
What do you mean here? What is the "final iteration"?

Reply:
Sorry if I'm unclear. We have significantly changed the scenario protocol for T5.1 and also the names and scenario settings are different. I would like to keep having access to the 'old' scenarios that have the 'old' names and that someone that did not upload those 'old' scenarios yet can still do so. Just for a while. I will request a removal of these scenarios when we do not need them anymore (i.e., in the final phase of T5.1, later this year).